ee/libc/include/ps2sdkapi.h

```diff
@@ -39,7 +40,10 @@ extern int (*_ps2sdk_closedir)(DIR *dir);
 #define PS2_CLOCKS_PER_MSEC (PS2_CLOCKS_PER_SEC / 1000) // 576
 
 typedef uint64_t ps2_clock_t;
-ps2_clock_t ps2_clock(void);
+static inline ps2_clock_t ps2_clock(void) {
+    // DEPRECATED VERSION USE INSTEAD GetTimerSystemTime
+    return (ps2_clock_t)GetTimerSystemTime();
+}
```

Comment:
This function should return BUSCLK/256 resolution: `return (ps2_clock_t)GetTimerSystemTime()>>8;`

Reply:
Yes I missed up between iGetTimerSystemTime and GetTimerSystemTime